### PR TITLE
New mobile menu

### PR DESCRIPTION
**Pull request by jkaeser**
I want a Probo build.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -5,7 +5,9 @@ gem "jekyll", "3.3.0"
 
 group :jekyll_plugins do
    gem 'jekyll-feed', '~> 0.6'
-   gem 'bundler', '~> 1.13.6'
+   gem 'bundler', '~> 1.13.7'
+   gem 'rack-jekyll', '0.5.0'
+   gem 'rake-jekyll', '1.1.0'
```
Comment (tortillaj) [inline Gemfile]: I don't think rake-jekyll and rack-jekyll are needed anymore. May need to test building the Jekyll site to be sure. They were added to enable deployment to GH pages, but we're not doing that anymore.

File: _stylesheets/01-utilities/_variables.scss [hunk 2]
```diff
@@ -40,7 +43,7 @@ $link-color-hover: $probo-blue;
 $probo-sitewidth: 1200px;
 $probo-menu-width: 300px;
 $border-radius: 4px;
-$header-height: 60px;
+$old-header-height: 60px;
```
Comment (tortillaj) [inline _stylesheets/01-utilities/_variables.scss]: Is this needed?

File: _stylesheets/02-atoms/_containers.scss [hunk 3]
```diff
@@ -7,6 +7,8 @@
 .wrapper {
   #sidebar-first {
     float: none;
+		display: none;
```
Comment (tortillaj) [inline _stylesheets/02-atoms/_containers.scss]: Weird alignment in this file. Unless Github is acting up.

File: _stylesheets/04-organisms/_section_landing.scss [hunk 4]
```diff
@@ -0,0 +1,34 @@
+.section-landing {
+
+  &__list {
+    list-style: none;
+    padding: 0;
+  }
+
+  &__list-item {
+  }
```
Comment (tortillaj) [inline _stylesheets/04-organisms/_section_landing.scss]: I detest empty Sass blocks :(

File: javascripts/src/mobile-nav.js [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    this.container.removeClass('is-revealed');
+    this.overlay.removeClass('main-menu__overlay');
+    this.menu.attr('aria-hidden', true);
+
+    this.menuIsOpen = false;
+  };
+
+  MobileMenu.prototype.toggleMenu = function() {
+    (this.menuIsOpen) ? this.closeMenu() : this.openMenu();
+  }
```
Comment (tortillaj) [inline javascripts/src/mobile-nav.js:62]: I added this toggleMenu method to simplify things. As a result, the implementation back in `probo.js` is simpler.

Doesn't need a change, just wanted to point it out.

File: gulpfile.js [hunk 6]
```diff
@@ -115,14 +117,12 @@ gulp.task('js', ['js:combine'], function () {
     .pipe(uglify({
       mangle: false,
       preserveComments: false,
-      output: {
-        beautify: true
```
Comment (tortillaj) [inline gulpfile.js:119]: Removing this setting lets the JS fully minify. Just pointing out the change I made.

**Review by tortillaj — CHANGES_REQUESTED** [review]

**Review by jkaeser — COMMENTED** [review]
Moved this comment elsewhere.

File: javascripts/src/mobile-nav.js (hunk 5 above)
Comment (jkaeser) [reply javascripts/src/mobile-nav.js:62]: We will need to duplicate these changes in https://github.com/ProboCI/probo.ci/pull/166.

Comment (tortillaj) [reply javascripts/src/mobile-nav.js:62]: I'll do it real quick.

File: _stylesheets/04-organisms/_section_landing.scss (hunk 4 above)
Comment (tortillaj) [reply _stylesheets/04-organisms/_section_landing.scss]: Removed.

File: _stylesheets/02-atoms/_containers.scss (hunk 3 above)
Comment (tortillaj) [reply _stylesheets/02-atoms/_containers.scss]: Re-aligned

File: _stylesheets/01-utilities/_variables.scss (hunk 2 above)
Comment (tortillaj) [reply _stylesheets/01-utilities/_variables.scss]: Deleted

File: Gemfile (hunk 1 above)
Comment (tortillaj) [reply Gemfile]: Removed

**Review by tortillaj — APPROVED** [review]
LGTM 🌟 🌟 🌟 🌟 🌟 🌟 🌟 🌟 🌟 🌟 🌟 🌟 🌟 🌟 

🌟 🌟 🌟 🌟 🌟 🌟 🌟 🌟 🌟 🌟

File: javascripts/src/mobile-nav.js (hunk 5 above)
Comment (tortillaj) [reply javascripts/src/mobile-nav.js:62]: That is done.

**Pull request merged**